### DISCUSSION TITLE
evaluation of oh-my-posh cannot be found

### DISCUSSION THREAD

**Discussion by louis-manabat** [JanDeDobbeleer/oh-my-posh#5198] [Troubleshoot] [answered]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

Working on my dotfiles, as I am migrating over to oh-my-posh (been loving it btw!) on both my personal servers and work Macbook, but when the eval command runs in .zshrc, I get hit with the (command not found: oh-my-posh). It does work though after I run "exec zsh" in the terminal, but I'm pretty sure I shouldn't be adding that to my dotfiles. I never ran into this issues on my servers which run Ubuntu.

Thanks!

### Theme

Personal/custom config

### What OS are you seeing the problem on?

macOS

### Which shell are you using?

zsh

### Log output

```Shell
Command doesn't get recognised in .zshrc :(
```

**Comment by JanDeDobbeleer** [comment]
@louis-manabat this is a `PATH` issue, not an oh-my-posh one. You need to make sure the location of the oh-my-posh executable is added to your `PATH` before you try to invoke it. Most likely your local `zsh` configuration doesn't do that until it loads the correct `rc` files when typing `exec zsh`, which usually is an indication of how your terminal starts `zsh` (e.g. as a login shell or not).

**Reply by louis-manabat** [reply, marked as answer]
Ah I found the solution - it seems that homebrew needs to be evaluated before that happens which is a bit confusing for me (still new to Mac). But at least I can start continuing to work on it!

Thanks for that!
